config/initializers/secure_headers.rb

```diff
@@ -5,25 +5,23 @@
   config.x_xss_protection = '1; mode=block'
   config.x_download_options = 'noopen'
   config.x_permitted_cross_domain_policies = 'none'
-  # rubocop:disable PercentStringArray
   config.csp = {
-    default_src: %w('self'),
+    default_src: ["'self'"],
     report_only: Rails.env.development? ? true : false,
-    frame_src: %w('self'), # deprecated in CSP 2.0
-    child_src: %w('self'), # CSP 2.0 only; replaces frame_src
+    frame_src: ["'self'"], # deprecated in CSP 2.0
+    child_src: ["'self'"], # CSP 2.0 only; replaces frame_src
     # frame_ancestors: %w('self'), # CSP 2.0 only; overriden by x_frame_options in some browsers
-    form_action: %w('self'), # CSP 2.0 only
+    form_action: ["'self'"], # CSP 2.0 only
     block_all_mixed_content: true, # CSP 2.0 only;
-    connect_src: %w('self'),
-    font_src: %w('self' data:),
-    img_src: %w('self' data:),
-    media_src: %w('self'),
-    object_src: %w('none'),
-    script_src: %w('self' *.newrelic.com *.nr-data.net),
-    style_src: %w('self'),
-    base_uri: %w('self')
+    connect_src: ["'self'"],
+    font_src: ["'self'", 'data:'],
+    img_src: ["'self'", 'data:'],
+    media_src: ["'self'"],
+    object_src: ["'none'"],
+    script_src: ["'self'", '*.newrelic.com', '*.nr-data.net'],
+    style_src: ["'self'"],
+    base_uri: ["'self'"]
   }
```

Comment:
I actually prefer the %w syntax. It makes it easier to add new entries without having commas and quotes.